Formal2024/Exercises/ex5.lean

```diff
@@ -77,6 +94,8 @@ theorem ne_types_of_different_property {α β : Type}
 
 example : ∃ α β : Type, α ≠ β := sorry
 
```

Comment:
To do this one, won't they need lemmas that say Nat is countable and R is uncountable? Or knowledge of the Empty type? Or am I missing an easier way to do it?

Reply:
It's supposed to make you think a little outside the box. So, if you don't know which types are countable or uncountable, then a quick search through `mathlib` will tell you that `\R` is uncountable and `\N` is countable

Reply:
But we didn't really discuss mathlib yet. This is a reminder that we should do it soon, but for the moment I'll keep explaining the new stuff we introduce

Reply:
In the week 5 material I will mention the `Empty` and `Unit` types. These can be used to answer this, because onle one of them is `Nonempty`.